### PR TITLE
Login read

### PR DESCRIPTION
**Pull request by j97b**
Login functionality is completed, when logged in the user should be a table of the boulders they own

### CODE REVIEW

File: src/main/resources/static/javaScript/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+                let tr = document.createElement('tr');
+                let td = document.createElement('td');
+                tr.setAttribute('id',user.id);
+                tr.setAttribute('class',"nameRows");
+                tr.addEventListener('click', () => {
+                    localStorage.setItem("userID",user.id);
+                    window.location = "html/bucketList.html";
+                });
+                let name = user.name.toString();
+                td.innerText = name.charAt(0).toUpperCase() + name.substring(1);
```
Comment (LukeCharles555) [inline src/main/resources/static/javaScript/index.js]: name.substring(1) => name.substring(1).toLowerCase()

Comment (LukeCharles555) [reply src/main/resources/static/javaScript/index.js]: signOut() change localStorage to sessionStorage
so users aren't logged in through multiple sessions

Comment (j97b) [reply src/main/resources/static/javaScript/index.js]: Did both of these

**Comment by LukeCharles555** [comment]
all ok :)

**Pull request merged**